### PR TITLE
Adds database associations to rails backend

### PR DESCRIPTION
**Pull request by GergKllai1**
# PT Link

[Adds database associations to rails backend](https://www.pivotaltracker.com/story/show/163781988)

# User Story

```
Fixes price table, connects session and user, adds price to backend methods
```

# Screenshots

screen shot 2019-02-06 at 21 49 42

screen shot 2019-02-06 at 21 49 55

# Description of PR

A small description of PR and functionality added

### CODE REVIEW

File: app/models/price_table.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 class PriceTable < ApplicationRecord
-    validates_presence_of :trainees, :price_point
+    validates_presence_of :price_point, :trainee_1, :trainee_3, :trainee_4, :trainee_5, :trainee_6
```
Comment (HannaSwDn) [inline app/models/price_table.rb]: Shouldn't we have 7 trainees at the most?

**Review by johannaanden — APPROVED** [review]
LGTM

File: app/models/price_table.rb (hunk 1 above)
Comment (kianaditya) [reply app/models/price_table.rb]: Discussed with Rupert and updated accordingly

**Review by HannaSwDn — APPROVED** [review]

**Review by magnus-thor — APPROVED** [review]
Good job

**Pull request merged**